ple/games/donkeykong/Game.py

```diff
@@ -0,0 +1,197 @@
+__author__ = 'Batchu Vishal'
+import pygame
+import sys
+from pygame.locals import *
+from Board import Board
+from .. import base
+
+'''
+This class defines the logic of the game and how player input is taken etc
+We run one instance of this class at the start of the game, and this instance manages the game for us.
+'''
+
+
+class DonkeyKong(base.Game):
+    def __init__(self):
+        '''
+        Set the height and width for the game element
+        FPS is set to 30 frames per second
+        '''
+        self.height = 520
+        self.width = 1200
+
+        actions = {
+                "left": K_a,
+                "right": K_d,
+                "jump": K_SPACE,
+                "up": K_w,
+                "down": K_s
+        }
+        base.Game.__init__(self, self.width, self.height, actions=actions)
+        self.init()
+
+    def init(self):
+        # Font is set to comic sans MS
+        self.myFont = pygame.font.SysFont("comicsansms", 30)
+
+        # Create a new instance of the Board class
+        self.newGame = Board(self.width, self.height)
+
+        # Initialize the fireball timer
+        self.fireballTimer = 0
+
+        # Assign groups from the Board instance that was created
+        self.playerGroup = self.newGame.playerGroup
+        self.wallGroup = self.newGame.wallGroup
+        self.ladderGroup = self.newGame.ladderGroup
+
+    def getScore(self):
+        return self.newGame.score
+
+    def game_over(self):
```

Comment:
Apologies: I thought there was a win criteria as we were checking for victory. If the game never ends, assuming the player doesn't lose lives, then that is fine!